includes/admin/events/metaboxes.php

```diff
@@ -624,6 +625,13 @@ function mdjm_event_metabox_options_playlist_row( $event_id )	{
 			'current'  => $enable_playlist ? 'Y' : 0,
 		) ); ?> <?php _e( 'Enable Playlist?', 'mobile-dj-manager' ); ?></p>
 
+    <p><?php _e ( 'Playlist Limit', 'mobile-dj-manager' ); ?>
+
+    <?php echo MDJM()->html->number( array(
+       		 	 'name'  => '_mdjm_event_playlist_limit',
+                	 'class' => '',
```

Comment:
The empty class can be removed from the array

Reply:
Aha yes - I wasn't sure if I'd succeeded in creating the postmeta in code properly.  For testing I manually manipulated the database & adding new events after that seemed to work ok

Reply:
The latest commit might just cover everything so far. Thanks!